README.md

```diff
@@ -30,6 +32,23 @@ API access is available only to users with active subscriptions. Visit https://a
 | `--[no-]parent-folder`   | None        | [Optional] Whether to download code into a project-named sub-folder. If true, downloads all project files directly to the specified directory. Defaults to `true`. |
 | `--[no-]as-module`   | None        | [Optional] Whether to generate the project as a Flutter module |
 
+## Deploy Firebase
+
+### Usage
```

Comment:
Let's also add a section about prerequisites: that you need to install and initialize `firebase-tools` and `npm`.